examples/nemo/eg3/kernels_managed_mem_trans.py

```diff
@@ -0,0 +1,195 @@
+# -----------------------------------------------------------------------------
```

Comment:
This file is very similar to kernels_trans.py. Could you please pull out the similar utility code into a separate file.

Reply:
Done.